test.py

```diff
@@ -683,8 +684,16 @@ class UnitTest(Test):
 
     def __init__(self, test_no: int, shortname: str, suite, args: str) -> None:
         super().__init__(test_no, shortname, suite)
-        self.path = path_to(self.mode, "test", suite.name, shortname.split('.')[0])
-        self.args = shlex.split(args) + UnitTest.standard_args
+        run_in_unshare = any(fnmatch.fnmatch(shortname, pattern)
+                             for pattern in suite.cfg.get("run_in_unshare", []))
+        cmd_args = []
+        if run_in_unshare:
+            cmd_args += ["unshare", "--map-root-user", "--net"]
```

Comment:
Actually, it will get broken in boost tests here https://github.com/scylladb/scylladb/pull/20597/files#diff-3665d65394f4f58a56a256ad6dd8621c68118d90fe56a19387e251c19cec2d2eL725
it will execute something like `unshare --run_test=blahblah ...`